README.md

```diff
@@ -201,6 +201,8 @@ trainer.fit(model, DataLoader(train), DataLoader(val))
 #### And without changing a single line of code, you could run on GPUs
 ```python
 # 8 GPUs
+from pytorch_lightning import Trainer
```

Comment:
can we simply add it to the code snippet above, because these lines here refer to the example above.

Reply:
ok I add it all import statements above.